morph/cli.py

```diff
@@ -139,14 +168,23 @@ def main():
     p_dump.add_argument('name', metavar='NAME', help='database to dump (all, known, ...)')
     p_dump.add_argument('--freq', action='store_true', help='include frequency as known to MorphMan')
 
+    def add_mizer(parser):
+        parser.add_argument('--mizer', default='mecab', choices=MIZERS.keys(),
+                            metavar='NAME',
+                            help='how to split morphemes (%s) (default %s)'
+                                 % (', '.join(MIZERS.keys()), 'mecab'))
+
     p_count = subparsers.add_parser('count', help='count morphemes in a corpus',
                         description='Count all morphemes in the given files and emit a frequency table.')
     p_count.set_defaults(action=cmd_count)
     p_count.add_argument('files', nargs='*', metavar='FILE', help='input files of text to morphemize')
-    p_count.add_argument('--mizer', default='mecab', choices=MIZERS.keys(),
-                         metavar='NAME',
-                         help='how to split morphemes (%s) (default %s)'
-                              % (', '.join(MIZERS.keys()), 'mecab'))
+    add_mizer(p_count)
+
+    p_next = subparsers.add_parser('next', help='find next notes to study from a corpus')
+    p_next.set_defaults(action=cmd_next)
+    p_next.add_argument('prio', metavar='FREQS', help='file of morphemes to study, with frequencies')
+    p_next.add_argument('notes', metavar='NOTES', help='file of newline-terminated notes, each tab-separated fields starting with the text')
```

Comment:
> If I were specifying a format from scratch I'd likely make some different choices, but this was already there so I picked it for consistency. Then again, given that you didn't recognize the format, maybe very few other users would either and there isn't much point in that consistency! We could do something different.

I'd like to hear your ideas. The POS and sub-POS thing is very mecab specific, and a remnant of the pre-morphemizer versions. Because I now see how all of this can be used (thanks to your example earlier), this won't hinder the merge.